thrust/internal/benchmark/bench.cu

```diff
@@ -48,13 +44,8 @@
 
 #define PP_CAT(a, b) a ## b
 
-// We don't use THRUST_NOEXCEPT because it's new, and we want this benchmark to
-// be backwards-compatible to older versions of Thrust.
-#if THRUST_CPP_DIALECT >= 2011
-  #define NOEXCEPT noexcept
-#else
-  #define NOEXCEPT throw()
-#endif
+#define NOEXCEPT noexcept
```

Comment:
Please just replace any instances of `NOEXCEPT` with `noexcept` and remove that macro